BrushUpCode/src/Coffee.java

```diff
@@ -0,0 +1,255 @@
+import java.util.Scanner;
+
+public class Coffee  {
+    private int coffeeBeans = 120;
+    private int water = 400;
+    private int milk = 540;
+    private int cups = 9;
+    private int dollar = 550;
+    State coffeeMachineState = State.NONE;
+    Scanner enter = new Scanner(System.in);
+    //constructor
+    public Coffee() {
+    }
+
+    //getters
+    private int getCoffeeBeans() {
+        return coffeeBeans;
+    }
+
+    private int getCups() {
+        return cups;
+    }
+
+    private int getDollar() {
+        return dollar;
+    }
+
+    private int getMilk() {
+        return milk;
+    }
+
+    private int getWater() {
+        return water;
+    }
+
+    //setters
+    private void setCoffeeBeans(int coffeeBeans) {
+        this.coffeeBeans = coffeeBeans;
+    }
+
+    private void setCups(int cups) {
+        this.cups = cups;
+    }
+
+    private void setDollar(int dollar) {
+        this.dollar = dollar;
+    }
+
+    private void setMilk(int milk) {
+        this.milk = milk;
+    }
+
+    private void setWater(int water) {
+        this.water = water;
+    }
+
+    //method that will be used in main (the only method), it contains methods fill take buy remaining in it
+    public void instruction(String input) {
+        this.coffeeMachineState = State.ACTION;
+        String take = "take";
+        String fill = "fill";
+        String buy = "buy";
+        String exit = "exit";
+        String remaining = "remaining";
+        while (!input.equalsIgnoreCase(exit)) {
+            if (input.equalsIgnoreCase(take)) {
+                take(dollar);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            } else if (input.equalsIgnoreCase(fill)) {
+                fill(water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            } else if (input.equalsIgnoreCase(buy)) {
+                buy(dollar, water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+
+            } else if (input.equalsIgnoreCase(remaining)) {
+                remaining(dollar, water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            }
+        }
+        System.out.println("thank you for using our services!");
+        this.coffeeMachineState = State.OFF;
+        System.exit(0);
+    }
+
+
+    //method for instruction take
+    private void take(int dollar) {
+        System.out.println("I gave you " + dollar + "$");
+        dollar -= dollar;
+        setDollar(dollar);
+
+    }
+
+    //method for instruction fill
```

Comment:
Try to use this type of java doc comments
http://www.drjava.org/docs/user/ch10.html#:~:text=In%20general%2C%20Javadoc%20comments%20are,method%20parameters%20or%20return%20values.

Reply:
done!